### PR TITLE
Use https instead of http as api address & implementation of editMonitor method

### PR DESCRIPTION
**Pull request by iler**
As the API works over https let's use that for better security.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -85,6 +85,22 @@ Client.prototype.newMonitor = function (options, callback) {
...
+    params.monitorID = options.monitorID;
+    if (options.FriendlyName) params.monitorFriendlyName = options.friendlyName;
+    if (options.url) params.monitorURL = options.url;
+    if (options.subType) params.monitorSubType = options.subType;
+    if (options.keywordType) params.monitorKeywordType = options.keywordType;
+    if (options.keywordValue) params.monitorKeywordValue = options.keywordValue;
+    if (options.httpUsername) params.monitorHTTPUsername = options.httpUsername;
+    if (options.httpPassword) params.monitorHTTPPassword = options.httpPassword;
+    if (options.alertContacts) params.alertcontacts = options.alertContacts.join('-');
+    if (options.interval) params.monitorInterval = options.interval;
```
Comment (ForbesLindesay) [inline index.js]: The indentation looks off here.  Why not just copy things across in the same style as the `newMonitor` method? `undefined` will be handled equivalently to not setting the property anyway.

**Comment by ForbesLindesay** [comment]
Other than that nit, this looks good :+1:

**Comment by iler** [comment]
Yes you are right. Will use the same style as in newMonitor. Will update it now! :)

**Comment by ForbesLindesay** [comment]
Awesome.  I've merged that and released as 1.1.0.

Would you like to be added as a collaborator on this project, to help review and merge pull requests?

**Comment by iler** [comment]
Why not. I may not always have time but will help when I have some to spare :)

**Comment by ForbesLindesay** [comment]
Thanks, you're added. Please follow these rules:

1. never commit directly to master unless you're only editing documentation, or you're in the process of cutting a new release.
2. never merge your own pull request unless it's had no comments from any collaborators for at least 24 hours.
3. If you cut a new release, make sure you follow [semver](http://semver.org/) when updating the version number.

If you let me know your npm username I'll add you as an owner so you can release new versions :)

**Pull request merged**